plugin/postgresql_test.go

```diff
@@ -0,0 +1,102 @@
+package plugin
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/vektra/cypress"
+	"github.com/vektra/neko"
+)
+
+func TestPostgresql(t *testing.T) {
+
+	n := neko.Start(t)
+
+	var db MockDBInterface
+
+	n.CheckMock(&db.Mock)
+
+	n.It("sets up a db", func() {
+		var p PostgreSQL
+		p.Init(&db)
+
+		db.On("Ping").Return(nil)
+		db.On("Exec", cCreateTable, []interface{}{}).Return(mock.Anything, nil)
+		db.On("Exec", cEnableHstore, []interface{}{}).Return(mock.Anything, nil)
```

Comment:
Removing that param all together did not work, but using `[]interface{}(nil)` solves that problem. Yay :confetti_ball: